src/screens/styles.ts

```diff
@@ -4,7 +4,103 @@ export const styles = StyleSheet.create({
   container: {
     flex: 1,
     flexDirection: 'column',
-    alignItems: 'center',
     backgroundColor: 'white',
+    padding: 10,
+  },
+  profileHeader: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    marginBottom: 8,
+    justifyContent: 'space-between',
+  },
+  profileName: {
+    fontWeight: 'bold',
+    fontSize: 16,
+    marginLeft: 8,
+  },
+  postDate: {
+    color: 'gray',
+    paddingLeft: 120,
+  },
+  postTextContainer: {
+    marginBottom: 10,
+  },
+  postText: {
+    fontSize: 14,
+    lineHeight: 20,
+  },
+  inlineLink: {
+    fontSize: 14,
+    color: 'black',
+  },
+  linkText: {
+    color: '#1E90FF',
+    textDecorationLine: 'underline',
+  },
+  postImage: {
+    width: '100%',
+    height: 200,
+    borderRadius: 10,
+    marginVertical: 10,
+  },
+  interactionRow: {
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    marginVertical: 10,
+    width: '100%',
+  },
+  icon: {
+    marginRight: 8,
+  },
+  likesText: {
+    marginLeft: 5,
+  },
+  commentSection: {
+    borderTopWidth: 1,
+    borderTopColor: '#ccc',
+    paddingTop: 10,
+    marginTop: 20,
+    paddingRight: 10,
```

Comment:
Avoid asymmetric spacing attributes. See if you can achieve this styling using flexbox properties.